src/main/resources/alma/common/item.xml

```diff
@@ -2,7 +2,7 @@
 <rules xmlns="http://www.culturegraph.org/metamorph">
   <entity name="hasItem[]" flushWith="record">
     <!-- ITM -->
-    <entity name="" reset="true" sameEntity="true">
+    <entity name="" reset="true" sameEntity="true" flushWith="ITM  .w">
```

Comment:
If subfield `"ITM  .w"` always exists within an ITM entry AND Also if  `"ITM  .w"` is always behind `"ITM  .x".  then this would be a good solution since not all ITMs have `"ITM  .c"`. Testdata looks like it.

Otherwise we would need another solution. 

I asked for it here: https://service-wiki.hbz-nrw.de/pages/viewpage.action?pageId=562135157

Reply:
I updated the workflow in a new commit.